src/Qowaiv.DomainModel/Aggregate_TAggregate_TId.cs

```diff
@@ -7,17 +7,17 @@
 /// <typeparam name="TId">
 /// The type of the identifier.
 /// </typeparam>
-public class AggregateRoot<TAggregate, TId> : AggregateRoot<TAggregate>
-    where TAggregate : AggregateRoot<TAggregate, TId>, new()
+public class Aggregate<TAggregate, TId> : Aggregate<TAggregate>
```

Comment:
In the summary it still mentions aggregate root

Reply:
In line 20 as well

Reply:
done!